ci-operator/config/openshift/etcd/openshift-etcd-origin-4.0-etcd-v3.3.10.yaml

```diff
@@ -0,0 +1,35 @@
+base_images:
```

Comment:
One problem with your branch names is that if you change anything about the branch your CI has to get recreated.  I would recommend creating `openshift-4.0` and tracking that here instead.